contracts/truefi2/LoanFactory2.sol

```diff
@@ -146,26 +146,6 @@ contract LoanFactory2 is ILoanFactory2, Initializable {
         return creditModel.rate(pool, borrowerScore, amount).add(fixedTermLoanAdjustment);
     }
 
-    /**
-     * @dev Deploy LoanToken with parameters
-     * @param _amount Amount to borrow
-     * @param _term Length of loan
-     */
-    function createLoanToken(
-        ITrueFiPool2 _pool,
-        uint256 _amount,
-        uint256 _term,
-        uint256 _maxApy
-    ) external override {
-        require(_amount > 0, "LoanFactory: Loans of amount 0, will not be approved");
-        require(_term > 0, "LoanFactory: Loans cannot have instantaneous term of repay");
-        require(poolFactory.isSupportedPool(_pool), "LoanFactory: Pool is not supported by PoolFactory");
-        uint256 apy = rate(_pool, msg.sender, _amount, _term);
-        require(apy <= _maxApy, "LoanFactory: Calculated apy is higher than max apy");
-
-        _createFTLALoanToken(_pool, msg.sender, _amount, _term, apy);
-    }
-
     function createFTLALoanToken(
```

Comment:
The only callers of `_createFTLALoanToken()` were `createLoanToken()`, and `createFTLALoanToken()`, so now we can collapse this into a single function.